RecoVertex/BeamSpotProducer/BuildFile.xml

```diff
@@ -16,6 +16,7 @@
 <use   name="clhep"/>
 <use   name="rootcore"/>
 <use   name="rootminuit2"/>
```

Comment:
I had to add `rootminuit` dependency to get the symbol for `TMinuitMinimizer`. I didn't check to see if `rootminuit2` can go away since I'm not certain why it was there to begin with.